src/js/plugin/PluginLoader.js

```diff
@@ -45,7 +48,9 @@ const PluginLoader = {
           PluginMountPoints: PluginMountPoints,
           pluginId: pluginId,
           React: React,
-          UIVersion: config.version
+          ajaxWrapper: ajaxWrapper,
+          config: Object.freeze(config),
+          PluginAPI: PluginAPI
```

Comment:
Do we really need to wrap these into a dedicate `PluginAPI` namespace? Shouldn't it be enough to simple provide `MarathonService` and  `MarathonActions`?